### PR TITLE
[MM-46439] Fix broken menu when switching to Boards

### PR DESCRIPTION
**Pull request by streamer45**
#### Summary

Boards were overriding the `.Menu` class and setting the position to `absolute` which would completely break the UI.
Adding a new e2e test to cover for this case.

#### Ticket

https://mattermost.atlassian.net/browse/MM-46439

### CODE REVIEW

**Review by cpoile — APPROVED** [review]
This isn't the first time that kind of thing has happened from Boards. It's one of the reasons Playbooks moved to styled components. :)

**Comment by streamer45** [comment]
> This isn't the first time that kind of thing has happened from Boards. It's one of the reasons Playbooks moved to styled components. :)

Nice. Looking forward to porting everything to that at some point :)

File: e2e/tests/start_call.spec.ts [hunk 1]
```diff
@@ -294,3 +294,30 @@ test.describe('setting audio output device', () => {
         }
     });
 });
+
+test.describe('switching products', () => {
+    test.use({storageState: userState.users[0].storageStatePath});
+
+    test.only('boards', async ({page}) => {
```
Comment (Phrynobatrachus) [inline e2e/tests/start_call.spec.ts:301]:
```suggestion
    test('boards', async ({page}) => {
```

Comment (streamer45) [reply e2e/tests/start_call.spec.ts:301]: Oh my, I always forget that, thanks 😓

**Comment by streamer45** [comment]
Looks like Boards is not installed/enabled on the CI installation. @Phrynobatrachus Is this something we could easily fix?

**Comment by Phrynobatrachus** [comment]
Oh right, yeah it should be an easy fix. I'll look at it this morning.

**Review by Phrynobatrachus — APPROVED** [review]

**Pull request merged**